aviary/subsystems/propulsion/test/test_propeller_performance.py

```diff
@@ -507,7 +508,7 @@ def test_mach(self):
             minimum_step=1e-12, abs_err_tol=5.0E-4, rel_err_tol=5.0E-5)
         assert_check_partials(partial_data, atol=1e-4, rtol=1e-4)
 
-    def test_tip_mach(self):
+    def tstest_tip_mach(self):
```

Comment:
is this renaming intentional? I think this means that testflo won't recognize it as a test any more

Reply:
I don't know what happened. It is named back now.